pyvistaqt/rwi.py

```diff
@@ -0,0 +1,771 @@
+# coding=utf-8
```

Comment:
Can you add a comment about where this was copied from, and under what license? Mayavi perhaps? IIRC Mayavi's RWI differs from that in the VTK repo... it would be nice to vendor here whichever parts of all RWIs work the best, to the extent that we can / know how. Some comment at the top of the file mentioning this, too, would help future maintenance and `diff`s with the other versions.

Reply:
... and as a side note, maybe having our own version of this will someday allow us to fix #20, which to me is actually a high-priority bug. There just hasn't been a "good" way to fix it so far because we have been stuck using VTK's version: https://gitlab.kitware.com/vtk/vtk/-/issues/17953

Or maybe if #20 is somehow magically fixed by PySide6, we could just recommend that everyone move to that ASAP!

Reply:
Whoops, I use the one from VTK actually :sweat_smile: but I can change it actually. I did not think about the hidpi issue right away

Reply:
I am not 100% sure which is more correct. It's possible that the VTK one *was* outdated but no longer is ... or I'm just misremembering which was better. I just know that last time I looked, they were not the same.

Hence why I think it should be investigated/documented clearly here for the next time we have to think about it :)